src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -35,22 +35,6 @@ public static ScriptSettingsManager Instance
 
         public bool IsDynamicSku => WebsiteSku == ScriptConstants.DynamicSku;
 
-        public virtual string AzureWebsiteDefaultSubdomain
```

Comment:
This was dead code. Noticed it when I was enforcing that no locations in the code use EnvironmentSettingNames.AzureWebsiteHostName directly anymore. All go through the provider now.